src/index.js

```diff
@@ -117,7 +118,7 @@ function reallyInitializeApp(routes) {
 export default function initializeApp(routes) {
   // set up logging when you're in production mode
   if (isProdMode()) {
-    Raven.config('https://e19420a2c46a4f97942553dfe8322cc4@sentry.io/1229723', {
+    Raven.config(process.env.SENTRY_DSN, {
```

Comment:
Did you confirm that the env vars are passed to/available in JS? I'm assuming it's python that loads the .env?

Reply:
Yes, I confirmed. I exported the envs and wrote a small javascript script to check if the var was accessible, and it worked. Yes, Python loads the envs.